index.js

```diff
@@ -102,9 +102,9 @@ module.exports = function (content) {
     dest: '',
     writeFiles: false,
     formatOptions: config.formatOptions || {
-        ttf: {
-            ts: 0 // it's important to set timestamp to fixed value for ttf generation to have consistent output in long term with the same hash
-        }
+      ttf: {
```

Comment:
Ok. Though I don't understand why you want to update the default value. Why don't you just add this in your config?